tools/opam-single-version-check.bash

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# makes sure no ordinary opam packages have more than a single version
```

Comment:
```
set -e pipefail
```

Reply:
The subshell that gather packages also needs to be moved outside of the for structure to fail gracefully.